dev-docs/publisher-api-reference.md

```diff
@@ -724,6 +725,65 @@ pbjs.addAdUnits({
 })
 ```
 
+<a name="adUnit-multiformat">
+
+##### Multi-format
+
+Multiple media formats may be declared on a single ad unit, allowing any bidder that supports at least one of those media formats to participate in the auction. Any bidder that isn't compatible with the specified `mediaTypes` will be dropped from the ad unit. If `mediaTypes` is not specified on an ad unit, `banner` is the assumed format and any banner bidder is eligible for inclusion.
+
+For examples of a multi-format ad units and behavior, see below.
+
+```javascript
+// each bidder supports at least one of the formats, so all will participate
+pbjs.addAdUnits({
+  code: 'div-banner-outstream-native',
+  mediaTypes: {
+    banner: {},
```

Comment:
Example should have at least one banner parameter.  Wouldn't the "banner" object be rejected in this case, since no size is specified?